### PR TITLE
Add description of HCC workflow

### PR DESCRIPTION
**Pull request by isc-etamarch**
Addresses #32

### CODE REVIEW

**Comment by isc-etamarch** [comment]
@isc-tleavitt Not sure if I got the merge request part of the HCC workflow correct.

File: docs/hcc.md [hunk 1]
```diff
@@ -0,0 +1,48 @@
+# git-source-control for Health Connect Cloud
+
+## Development Workflow
+
+git-source-control has two main use modes: Basic and Expert. For Health Connect Cloud, we recommend beginning with basic mode, since it includes all of the necessary functionality for proper source control development while being easy to use for those with little development / source control experience. The recommended workflow for HCC is to create featyre branches, merge them into development, and then git pull the changes in the shared development namespace. The rest of this section explains this workflow more in depth for those with less source control experience. 
+
+### Feature branches
```
Comment (isc-tleavitt) [inline docs/hcc.md:7]: We came up with the term "Interface branch" in a different discussion you may or may not have been part of; let's use that.

File: docs/hcc.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+![Creating a new via sidebar](images/hcc/newbranch.png)
+![Naming new branch](images/hcc/newbranchnaming.png)
+
+Finally, you should click on the new branch you just made in the sidebar, and then press "Push".
+
+![Push your new branch](images/hcc/pushbranch.png)
+
+This makes sure that your branch will be tracked in Gitlab, so that other users can checkout your branch to test your changes on their own namespaces.
+
+After this, you can start working on the change.
```
Comment (isc-tleavitt) [inline docs/hcc.md:26]: This is more of an expert mode workflow, really. I'm realizing that you might not have actually had enough context on basic mode to talk through the workflow. None of this is wrong, it's just too complicated for what we want to start with for someone with zero git background.

I'd honestly be fine with repurposing all you have as the expert mode documentation / as part of the coverage of Git WebUI. This is largely useful content still, just not for the bare-bones simplest-possible Health Connect Cloud use case.

**Review by isc-tleavitt — CHANGES_REQUESTED** [review]
A few initial points of feedback - this is all accurate for more of an expert mode / Git WebUI-based workflow, but not actually the starting point for the Health Connect Cloud workflow.

**Review by isc-tleavitt — APPROVED** [review]
Just going to merge these because they're an incremental improvement; we can keep iterating.

**Pull request merged**